Userland/Libraries/LibWeb/DOM/EventTarget.cpp

```diff
@@ -6,6 +6,7 @@
  */
 
 #include <AK/StringBuilder.h>
+#include <Kernel/API/KeyCode.h>
```

Comment:
Please don't include SerenityOS kernel headers in LibWeb DOM code :)

If we need an `enum` for this, we should put a platform independent somewhere in LibWeb.

Reply:
I saw this was already included a few times in LibWeb, so i assumed it was OK to use more.. 
Is it spesifically just for the DOM subfolder?

Reply:
Oh whoops! No, I'd say it's poor form to use kernel headers anywhere in LibWeb unless we're making syscalls. :)
But since it was already done elsewhere, maybe it's fine to do one more in this PR and we can clean it up globally in a separate effort.

Reply:
Oh, and you have 32 commits in this PR branch ;)

Reply:
Should be ok now :)